### PR TITLE
User documentation on RTD

### PR DESCRIPTION
**Pull request by prayerslayer**
[Preview](http://zappr.readthedocs.org/en/78-docs)

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+site_name: Zappr
+site_favicon: img/favicon.ico
+repo_url: https://github.com/zalando/zappr
+docs_dir: docs
+pages:
+  - Overview: index.md
+  - Setup: setup.md
+  - FAQ: faq.md
```
Comment (mfellner) [inline mkdocs.yml]: Should we have a newline at the end?

File: docs/setup.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Using your GitHub account, sign in to Zappr [here](https://zappr.opensource.zalan.do/login).
+
+![Login](img/setup/login.png)
+
+Authorize Zappr. We outline why we need certain scopes in our [FAQ](https://zappr.readthedocs.org/en/latest/faq).
+
+Once you're back at Zappr you will see all repositories that we know listed on the left. Initially we only fetch the first couple of repositories, so if you don't find the one you're looking for please click the blue "Sync with Github" button. It will then load all the repositories, which might take a couple of seconds. Also use this button if you want to enable Zappr for repositories that are new in your account.
+
+![IMAGE](img/setup/repo-list.png)
```
Comment (mfellner) [inline docs/setup.md]: Would be nice if we could make the images a bit smaller!

Comment (prayerslayer) [inline docs/setup.md]: I added a media query

**Comment by mfellner** [comment]
:+1:

**Pull request merged**